### ISSUE TITLE
[Accessibility] Field cannot be labeled

### ISSUE THREAD

**Issue by almedina-ms** [xceedsoftware/wpftoolkit#1594]
Using the AutomationProperties.SetLabeledBy or IsRequiredForForm doesn't seem to make an screen reader get the hints for the input field. 

https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationproperties.labeledbyproperty?view=netcore-3.1

Once the element get focus, it just reads "edit"

**Comment by XceedBoucherS** [comment]
Hi,
Are you using the Plus version of the Toolkit or the Free version ?
UI Automation is only available in the "Plus" version : https://github.com/xceedsoftware/wpftoolkit/wiki/Xceed-Toolkit-Plus-for-WPF

Thank you
